docs/using.rst

```diff
@@ -8,14 +8,16 @@ User Guide
 Getting Certbot
 ===============
 
-To get specific instructions for installing Certbot on your OS, we recommend
-visiting certbot.eff.org_. If you're offline, you can find some general
+To get specific instructions for installing Certbot on your OS,
+visit certbot.eff.org_. This is the easiest way to install Certbot. 
+
+If you're offline, or if your webserver or OS are not in the menu, you can find some general
```

Comment:
Ideally, everyone's webserver or OS should be in the menu. We have a catch all for webserver titled "None of the above". While we have "Other UNIX", we don't say anything about non-UNIX systems on the site. I created certbot/website#147 to track that issue.

With this in mind, I think this line should remain "If you're offline, you can find some general..."